invenio_records_marc21/config.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# Invenio-Records-Marc21 is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Default configuration."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_indexer.api import RecordIndexer
+from invenio_records_rest.utils import allow_all, check_elasticsearch
+from invenio_search import RecordsSearch
+
+RECORDS_REST_ENDPOINTS = {
+    "recid": dict(
```

Comment:
needs to be changed - to be discussed. 
NO CHANGE requires.
```suggestion
    "marcid": dict(
```